dim/spec/dim/raw_rst_spec.rb

```diff
@@ -0,0 +1,365 @@
+# frozen_string_literal: true
+
+require 'dim/globals'
+require 'dim/loader'
+require 'dim/exporter/raw_rst'
+
+RSpec.describe Dim::RawRST do
+  subject(:raw_rst) { described_class.new(loader) }
+
+  let(:loader) do
+    instance_double(Dim::Loader)
+  end
+
+  describe '.file_extension' do
+    subject(:method_call) { described_class.file_extension }
+
+    it 'returns "rst"' do
+      expect(method_call).to eq('rst')
+    end
+  end
+
+  describe '#initialize' do
+    subject(:method_call) { raw_rst }
+
+    it 'sets hasIndex to true' do
+      expect(raw_rst.hasIndex).to be(true)
+    end
+  end
+
+  describe '#index' do
+    subject(:method_call) { raw_rst.index(buffer, category, origin, modules) }
+
+    let(:buffer) { StringIO.new }
+    let(:origin) { 'ESR Labs' }
```

Comment:
This is just a test, so maybe use a fictional company name and simplified requirement texts.

Reply:
Also changed. Removed replaced all of them for "ACME Inc."